docs/app-gen.md

```diff
@@ -0,0 +1,118 @@
+# app-gen
+## Overview
+app-gen is an experimental application generation utility for OpenShift v3. It is used to create the necessary configuration to build and deploy code from a source repository. 
+
+## Usage
+```
+Generate a new application in OpenShift from source code.
+
+Usage:
+openshift-experimental [source]
+
+The source parameter may be a directory or a repository URL. 
+If not specified, the current directory is used.
+
+Examples:
```

Comment:
This is dependent on the command-line utility output, which is what I copied to this document. If I change them in the document, they won't match what you get from the command line. I've added a remote repo url.